cmake/modules/add_python_tests.cmake

```diff
@@ -37,15 +37,26 @@ function(add_python_tests)
     find_package(Python3 REQUIRED COMPONENTS Interpreter)
   endif()
 
-  # Get the list of tests from pytest - force rootdir to be the working directory
+  # Set rootdir to the working directory to limit the search, and for relative test names
+  # Set confcutdir to the working directory to ignore top level conftest.py
+  set(PYTEST_IGNORE_ROOT_STR "--rootdir=${PYTEST_WORKING_DIRECTORY} --confcutdir=${PYTEST_WORKING_DIRECTORY}")
+
+  # Get the list of tests from pytest
+  set(pytest_collect_cmd "${Python3_EXECUTABLE} -m pytest ${PYTEST_WORKING_DIRECTORY}/${PYTEST_INPUT} --collect-only ${PYTEST_IGNORE_ROOT_STR}")
```

Comment:
What should `PYTEST_INPUT` be here? Could help to add a function docstring

Reply:
https://github.com/nvidia-holoscan/holohub/pull/952/files#diff-fa0680c59bedfdb7963e5cab80aab5f157f8ce1ab864dc906ca4312a395510d7R17